scripts/lint_feedstocks.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env conda-execute
+
+# conda execute
+# env:
+#  - python
+#  - conda-smithy
+#  - gitpython
+#  - pygithub
+# channels:
+#  - conda-forge
+# run_with: python
+
+import os
+import argparse
+import textwrap
+import random
+import re
+
+import git
+import github
+
+import conda_smithy.github
+import conda_smithy.lint_recipe
+import conda_smithy
+
+import conda_smithy.feedstocks as feedstocks
+
+
+parser = argparse.ArgumentParser(description=('Lint all of the feedstocks, '
+                                              'raising issues if there is any.'))
+parser.add_argument('--feedstocks-dir', help="The location of the feedstocks.",
+                    default="~/dev/conda-forge/feedstocks")
+parser.add_argument('--regexp', help="Regexp of feedstocks to consider.",
+                    default=".*")
+args = parser.parse_args()
+
+feedstocks_dir = os.path.expanduser(args.feedstocks_dir)
+feedstocks.clone_all('conda-forge', feedstocks_dir)
+feedstocks.fetch_feedstocks(feedstocks_dir)
+
+regexp = re.compile(args.regexp)
+randomised_feedstocks = [feedstock for feedstock in feedstocks.cloned_feedstocks(feedstocks_dir)
+                         if regexp.match(feedstock.package)]
+random.shuffle(randomised_feedstocks)
```

Comment:
At one point I had a `limit` parameter which only did the first N feedstocks. I didn't want `aardvark-feedstock` to be better represented than `zebra-feedstock` :wink: